v0/destinations/facebook_pixel/transform.js

```diff
@@ -372,7 +435,16 @@ const responseBuilderSimple = (
         .map(String)
         .join(",");
     } else if (typeof customData.content_category === "object") {
-      throw new CustomError("Category must be must be a string");
+      throw new ErrorBuilder()
+        .setMessage("Category must be a string")
```

Comment:
It'd be good if we can point out which field in the source event this category maps to.

Reply:
It looks like it is a little tricky to get the category map information(from the source-code)

Reply:
It's from the `properties.category`.
Also, as per the logic, the `content_category` field can either be a string or an array. So, let's handle that too.

Reply:
Refactored the error handling for this part to their specific methods. And changed the error message. Please check and do the needful